### PR TITLE
MGMT-10085: Get real filepath of RAID device

### PR DESCRIPTION
**Pull request by javipolo**
When having LVM on top of software raid, the name of the device reported by mdadm is a symlink to the real block device in /dev, causing problems identifying which LVM volume groups need to be removed before cleaning the RAID

This should fix that issue

### CODE REVIEW

File: src/ops/ops.go [hunk 1]
```diff
@@ -510,7 +510,12 @@ func (o *ops) getRaidDevices2Members() (map[string][]string, error) {
 		}
 
 		fields := strings.Fields(lines[i])
-		raidDeviceName := fields[1]
+		// In case of symlink, get real file path
+		raidDeviceName, err := filepath.EvalSymlinks(fields[1])
+		if err != nil {
+			return nil, err
```
Comment (omertuc) [inline src/ops/ops.go:516]: There should be much more of this in the file but let's at-least make sure new code has it - errors should be given context, otherwise it's hard to follow where they come from

```suggestion
			return nil, errors.Wrapf(err, "failed to evaluate raid symlink")
```

Comment (javipolo) [reply src/ops/ops.go]: sure!
Just added your suggestion :)
